src/NexusMods.App.UI/Controls/DevelopmentBuildBanner/DevelopmentBuildBannerView.axaml.cs

```diff
@@ -22,12 +22,11 @@ public DevelopmentBuildBannerView()
         InitializeComponent();
 
         var appVersion = GetAppVersion();
-        Text = $"{appVersion} - DEVELOPMENT USE ONLY";
+        Text = $"{appVersion} - Stardew Valley Preview Build";
```

Comment:
It should still say `DEVELOPMENT USE ONLY` if we're in debug mode:

```suggestion
        var suffix = CompileConstants.IsDebug ? "DEVELOPMENT USE ONLY" : "Stardew Valley Preview Build";
        Text = $"{appVersion} - {suffix}";
```